app.js

```diff
@@ -1,21 +1,54 @@
+/* eslint-disable global-require */
+/* eslint-disable max-len */
 require('./util/env').configure(); // configure the environment variables
 require('./util/mongodb'); // configure mongoose db connection
 require('./util/aedes'); // configure MQTT broker
 require('./util/s3'); // configure s3 client
+const https = require('https');
```

Comment:
```suggestion
const https = (process.env.PRODUCTION) ? require('https') : {};
```
Do not require module unless needed